src/Form/matching.js

```diff
@@ -515,7 +501,9 @@ class Matching {
                 matchers: {}
             },
             'cssSelector': {
-                selectors: {}
+                selectors: {
+                    FORM_INPUTS_SELECTOR: FORM_INPUTS_SELECTOR
```

Comment:
This can be simplified with the shorthand. But can you explain why this is needed now but wasn't needed before?

Reply:
it's because some of our unit tests are more comprehensive now and they cause a code-path to execute that requires this.